src/ui/components/Notice/index.js

```diff
@@ -22,11 +23,9 @@ type Props = {
  * A Photon style notification bar.
  *
  * See https://design.firefox.com/photon/components/message-bars.html
- *
- * TODO: implement generic notices and warnings.
  */
 const Notice = ({
-  action, actionText, children, className, type,
+  action, actionText, children, className, type = 'generic',
```

Comment:
I don't think it's a good idea to default `type` to `generic`. I'd rather the caller see an error when they forget to declare a `type`. If we find that it's cumbersome to declare an explicit `type` all the time then we could revisit it. However, I think `error` would be a more common type.